addons/project/static/src/components/subtask_one2many_field/subtask_list_renderer.js

```diff
@@ -18,3 +19,5 @@ export class SubtaskListRenderer extends ListRenderer {
         });
     }
 }
+
+SubtaskListRenderer.components = { ...SubtaskListRenderer.components, ViewButton: TaskViewButton };
```

Comment:
Same here, it can be static inside the class.
`SubtaskListRenderer.components.ViewButton = TaskViewButton;` is also possible, but static is better.

Reply:
Hello @auon-odoo 
Changed are applied. Thank You :slightly_smiling_face: